xml/System.Diagnostics/Activity.xml

```diff
@@ -612,7 +621,7 @@ An ID has a hierarchical structure: `root-id.id1_id2.id3_`. The ID is generated
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Indicates if the this Activity object should be populated with all the propagation info and also all other properties such as Links, Tags, and Events.</summary>
```

Comment:
>summary [](start = 9, length = 7)

could you please add a remark section saying: 

This property is just indication about the purpose of the Activity object but it doesn't affect any functionality like AddTag, AddEvent, AddBaggage...etc. 

CC @noahfalk